src/Formula/Parsing/Delayed.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE DeriveGeneric #-}
+module Formula.Parsing.Delayed where
+
+
+import Text.Parsec
+import Text.Parsec.String (Parser)
+import ParsingHelpers (fully)
+
+import Control.Monad.Output (LangM, english, german, OutputMonad)
+
+import LogicTasks.Helpers (reject)
+
+import Data.Typeable (Typeable)
+import GHC.Generics (Generic)
+
+newtype Delayed a = Delayed String deriving (Eq, Show, Typeable, Generic)
+
+delayed :: String -> Delayed a
+delayed = Delayed
+
+parseDelayed :: Delayed a -> Parser a -> Either ParseError a
+parseDelayed (Delayed str) p = parse p "(delayed input)" str
```

Comment:
Taucht dieses `"(delayed input)"` dann eigentlich in den anzuzeigenden Fehlermeldungen auf? Für die Studierenden wäre der Begriff sicher verwirrend.

Reply:
Ja tut es. In erster Linie wollte ich damit das Debuggen von Parserfehlern einfacher machen. Kann schon sein, dass das verwirrend ist. Ich denke aber, dass es erkennbar sein sollte, dass der Fehler aus dem verzögerten Parsen entsteht (evtl sogar aus welcher Phase genau, bei mehreren?). Vieleicht ist eine Formulierung besser, die Studierende eher als "neutrales Element" im Fehlerstring lesen. Evtl. "(task-parser-input)" oder "(answer string)"?

Reply:
Wahrscheinlich ist `"(answer string)"` ganz gut.